src/boot/index.js

```diff
@@ -21,6 +21,18 @@ if (isBrowserSupported()) {
   const config = /** @type {AnnotatorConfig|SidebarAppConfig} */ (
     parseJsonConfig(document)
   );
+
+  // When the boot script is executed from the browser extension, at least one
+  // config is required
+  const isExtensionContext = !!(
+    /** @type {any} */ (window).chrome?.runtime?.id
+  );
+  if (!Object.keys(config).length && isExtensionContext) {
+    throw new Error(
+      'Could not bootstrap hypothesis as there is no configuration'
```

Comment:
```suggestion
      'Could not start Hypothesis extension as configuration is missing'
```

Tweaked this wording to capitalize Hypothesis, mention the extension specifically, and prefer _start_ or _launch_ as probably more meaningful than "bootstrap" for users rather than developers of Hypothesis.